src/SessionSetup/ConnectToTargetDialog.ui

```diff
@@ -24,7 +24,7 @@
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
-    <enum>QLayout::SetFixedSize</enum>
+    <enum>QLayout::SetDefaultConstraint</enum>
```

Comment:
Intended change?

Reply:
Yes, without this change the dialog layout is broken when a instance name is displayed instead of an ID (because IDs are much longer, and the dialog was auto-resizing...).